packages/event-client/src/service-bus-event-client.js

```diff
@@ -1,6 +1,10 @@
 import { DefaultAzureCredential } from '@azure/identity';
 import { ServiceBusClient } from '@azure/service-bus';
 
+// 100 events would allow each event to have a body size of 10kb which is more than enough
+// I haven't seen an event in the test environment larger than 1kb
+const eventChunkSize = 100;
```

Comment:
I'm wondering if this logic should live where the event-client is used instead of in this package. Then the application has control over chunk size and failure handling. With this implementation you don't know which set of events failed to send. Any thoughts?

Reply:
You're right - hadn't thought about failures. I configured it this way because adding an additional parameter would be difficult since we aren't using an object and there's already a nullable parameter at the end of the parameter list. I'll take this logic out of the client entirely and let the user manage this instead.

Maybe a max events array size is a better idea, and we throw an error?